Libraries/TranscriptTestRunner/TestRunner.cs

```diff
@@ -36,11 +37,13 @@ public class TestRunner
         /// </summary>
         /// <param name="client">Test client to use.</param>
         /// <param name="replyTimeout">The timeout for waiting for replies (in seconds). Default is 180.</param>
+        /// <param name="thinkTime">The timeout think time before sending messages to the bot (in miliseconds). Default is 0.</param>
         /// <param name="logger">Optional. Instance of <see cref="ILogger"/> to use.</param>
-        public TestRunner(TestClientBase client, int replyTimeout = 180, ILogger logger = null)
+        public TestRunner(TestClientBase client, int replyTimeout = 180, double thinkTime = 0, ILogger logger = null)

```

Comment:
Any reason for this to be double? shouldn't it be int or long?

Reply:
double is the value that the TimeSpan.fromMiliSeconds method take, so I thought it would be the best